src/components/GameResultTable.js

```diff
@@ -1,142 +1,39 @@
-import { RegionalZonaRcGameScores } from '../games/RegionalZonaRcGameScores';
-import { Levante124GameScores } from '../games/Levante124GameScores';
-import { MiniCrawlerPassionGameScores } from '../games/MiniCrawlerPassionGameScores';
-import { GenericGameScores } from '../games/GenericGameScores';
-import { useTranslation } from 'react-i18next';
-import { GAME_TYPE_ISRCC,
-    GAME_TYPE_LEVANTE,
-    GAME_TYPE_COPAESPANA,
-    GAME_TYPE_MINICRAWLERPASSION,
-    GAME_TYPE_GENERIC,
-    GameUtils } from '../games/Game';
-
-import fiascoIcon from '../resources/img/iconFiasco.png';
-import winnerIcon from '../resources/img/iconWinner.png';
-import batteryIcon from '../resources/img/iconBattery.png';
-import openIcon from '../resources/img/arrowdown.png';
-
+import { GameResultTableHeader } from './GameResultTableHeader';
+import { GameResultTablePlayerHeader } from './GameResultTablePlayerHeader';
+import { GameResultTablePointsRow } from './GameResultTablePointsRow';
+import { GameResultTableZonePointsRow } from './GameResultTableZonePointsRow'
+import { getGamePlayerResultIcon } from '../modules/gamePlayer/GamePlayerUtils';
 
 function GameResultTable({game, isDraw}) {
-    const { t } = useTranslation(['main']),
-        players = [];
-    let gameTypeTexts = Utils.tokenToTexts(t,AecarGameScores.texts),
-        j=0, i=0;
-
-    switch(game.gameType) {
-        case GAME_TYPE_ISRCC:
-            gameTypeTexts = Utils.tokenToTexts(t,IsrccGameScores.texts);
-            break;
-        case GAME_TYPE_LEVANTE:
-            gameTypeTexts = Utils.tokenToTexts(t,Levante124GameScores.texts);
-            break;
-        case GAME_TYPE_COPAESPANA:
-            gameTypeTexts = Utils.tokenToTexts(t,RegionalZonaRcGameScores.texts);
-            break;
-        case GAME_TYPE_MINICRAWLERPASSION:
-            gameTypeTexts = Utils.tokenToTexts(t,MiniCrawlerPassionGameScores.texts);
-                break;
-        case GAME_TYPE_GENERIC:
-            gameTypeTexts = Utils.tokenToTexts(t,GenericGameScores.texts);
-                break;        
-        default:
-            gameTypeTexts = Utils.tokenToTexts(t,AecarGameScores.texts);
-            break;
-    }
-
-    function onClickZone(event) {
-        let element = event.target;
-
-        element = (element.tagName === 'IMG' ? element.parentElement : element);
+    const players = [];
+    let j=0, i=0;
 
-        const zoneTr = element.parentElement.nextElementSibling;
-        zoneTr.classList.toggle("closed");
-    }
-
-    players.push(<tr key={`game${new Date().getMilliseconds()}`}>
-        <td></td>
-        <td className=""></td>
-        <td>{t("description.puntos")}</td>
-        <td>{t("description.puertas")}</td>
-        <td>{t("description.bonificacion")}</td>
-        <td>{t("description.puntos")} {t("description.portiempo")}</td>
-        <td></td>
-    </tr>);
-    
+    players.push(<GameResultTableHeader></GameResultTableHeader>);
```

Comment:
keys are not managed here. Get them from the individual components that will remove some props